docs/nx/next/init-deployment.md

```diff
@@ -87,38 +90,40 @@ As a rule of thumb, each task here references a target execution via Nx defined
 
 ```yaml
 helm:
-    description: Lint Helm Charts
-    command:
-      - npx nx affected --base="$BASE_SHA" --target=helm-lint
+  description: Lint Helm Charts
+  command:
+    - npx nx affected --base="$BASE_SHA" --target=lint
 ```
 
 `apps/myapp/project.json`
 
 ```yaml
-"helm-lint":
+"lint":
   {
     "executor": "nx:run-commands",
     "options":
       {
-        "commands": [{ "command": "helm lint", "forwardAllArgs": false }],
+        "commands": [{ "command": "lint", "forwardAllArgs": false }],
```

Comment:
It's still running the helm-lint command, but please double check

Reply:
command is `helm lint` yes, under target `lint` - updated